test/features/icons/icons.test.js

```diff
@@ -1,45 +1,51 @@
 import { readFile } from '@web/test-runner-commands';
 import { expect } from '@esm-bundle/chai';
 import { stub } from 'sinon';
-import { setConfig, getConfig, createTag } from '../../../libs/utils/utils.js';
+import { waitForElement } from '../../helpers/waitfor.js';
 
-const { default: loadIcons } = await import('../../../libs/features/icons/icons.js');
-
-const codeRoot = '/libs';
-const conf = { codeRoot };
-setConfig(conf);
-const config = getConfig();
+const { default: loadIcons, setNodeIndexClass } = await import('../../../libs/features/icons/icons.js');
+const mockRes = ({ payload, status = 200, ok = true } = {}) => new Promise((resolve) => {
+  resolve({
+    status,
+    ok,
+    json: () => payload,
+    text: () => payload,
+  });
+});
 
 document.body.innerHTML = await readFile({ path: './mocks/body.html' });
 
 let icons;
+const svgEx = `<?xml version="1.0" encoding="UTF-8"?>
+<svg id="arrow-down" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
+  <path fill="currentcolor" d="M12,10V1.5a.5.5,0,0,0-.5-.5h-5a.5.5,0,0,0-.5.5V10H2.5035a.25.25,0,0,0-.177.427L9,17.1l6.673-6.673A.25.25,0,0,0,15.4965,10Z"></path>
+</svg>`;
 
 describe('Icon Suppprt', () => {
-  let paramsGetStub;
-
-  before(() => {
-    paramsGetStub = stub(URLSearchParams.prototype, 'get');
-    paramsGetStub.withArgs('cache').returns('off');
+  beforeEach(() => {
+    stub(window, 'fetch').callsFake(() => mockRes({}));
   });
 
-  after(() => {
-    paramsGetStub.restore();
+  afterEach(() => {
+    window.fetch.restore();
```

Comment:
You could even do `sinon.restore();` which should handle any stubs without having to explicitly name them.